src/utilities.rs

```diff
@@ -0,0 +1,18 @@
+use std::path::PathBuf;
+
+pub(crate) fn check_sound(sound_name: &str) -> bool {
+    dirs::home_dir()
+        .map(|path| path.join("/Library/Sounds/"))
+        .into_iter()
+        .chain(
+            [
+                "/Library/Sounds/",
+                "/Network/Library/Sounds/",
+                "/System/Library/Sounds/",
+            ]
+            .iter()
+            .map(PathBuf::from),
+        )
+        .map(|sound_path| sound_path.join(format!("{}.aiff", sound_name)))
+        .any(|some_path| some_path.exists())
+}
```

Comment:
why move this out?
is somebody else using it?
are we testing it?

Reply:
I use `check_sound` in the `data.rs` (or `notification.rs`) file, but it felt wrong to put it in it since it wasn't really related to the actual notification. I didn't want to put it in `lib.rs` either because I felt it was kind of "polluting" that file since no part of `lib.rs` needs it.

I guess I can just put it in the data/notification file.

Reply:
fair point, was just asking